zappa/cli.py

```diff
@@ -1871,6 +1871,7 @@ def init(self, settings_file="zappa_settings.json"):
                 "s3_bucket": bucket,
                 "runtime": get_venv_from_python_version(),
                 "project_name": self.get_project_name(),
+                "exclude": ["boto3", "dateutil", "botocore", "s3transfer", "concurrent"],
```

Comment:
Ok, so on initilization, the sample zappa_settings_file will include the excludes, but if excludes are removed from the settings, then none will be included by default, right?

If my understanding is correct, I think this is a great "fix" in how excludes work.

I believe this is mentioned in the comments, but this merge should wait until we're ready to issue another major release.

Reply:
Yes, that's right.
Since boto3 etc. are no longer excluded for users who do not have an exclude in zappa_settings,
So it's good to deploy on the next major release. (ex. 0.58.0)